doc/fluidsynth-v20-devdoc.txt

```diff
@@ -72,6 +72,7 @@ What is FluidSynth?
 
 \section NewIn2_2_0 What's new in 2.2.0?
 
+- #fluid_file_callbacks_t now uses <code>long long</code> as file-offset type (see #fluid_long_long_t). This is a breaking change, which allows to load SoundFonts bigger than 2GiB on Windows. This change required to bump fluidsynth's SOVERSION.
 - The sequencer has received a major revisal. For you that means:
   - The sequencer's queue no longer blocks the synthesizer thread, due to being busy arranging its events internally.
   - Eevents that share the same tick was given a new, documented order, see fluid_sequencer_send_at().
```

Comment:
>Eevents ...

A typo.

>tick was given a new, documented order, see fluid_sequencer_send_at():

1)FLUID_SEQ_SYSTEMRESET events precede any other event type.
2)FLUID_SEQ_UNREGISTERING events succeed FLUID_SEQ_SYSTEMRESET and precede other event type.
3)FLUID_SEQ_NOTEON and FLUID_SEQ_NOTE events succeed any other event type.
4)Otherwise `the order is undefined`. 


Please, for case 4, it means the reader should understand that for other events `the order is left unchanged`. Is not it ?

Reply:
> Please, for case 4, it means the reader should understand that for other events the order is left unchanged. Is not it ?

No. It's undefined. Under the hood, we now use heap sort, which is not stable. Thus, the order will probably change. Marcus and I had a few discussions about that in #604.

Thx for pointing out the typo. Will fix it on master.

Reply:
>Thus, the order will probably change.

Thx, for this useful precision.